integrations/azure-devops/azure_devops/client/azure_devops_client.py

```diff
@@ -258,6 +258,49 @@ async def get_repository(self, repository_id: str) -> dict[Any, Any]:
         repository_data = response.json()
         return repository_data
 
+    async def get_columns(self) -> AsyncGenerator[list[dict[str, Any]], None]:
+        async for boards in self.get_boards_in_organization():
+            for board in boards:
+                yield [
+                    {
+                        **column,
+                        "__board_id": board.get("id"),
```

Comment:
return the full board data instead of just the id

Reply:
Done